http/src/main/resources/reference.conf

```diff
@@ -31,6 +31,7 @@ dataproc {
     workerDiskSize = 100
     numberOfWorkerLocalSSDs = 0
     numberOfPreemptibleWorkers = 0
+    zone = "us-central1-a"
```

Comment:
`zone` and `region` actually exist already up above.

This will actually change behavior: it will cause all Dataproc clusters to be created in the `us-central1-a` zone whereas currently they are created in the `us-central1` region (zones a through f). 

I'm thinking we should still use region for Dataproc clusters and zone for GCE.

Reply:
Actually maybe `zone` and `region` on L15-16 should be removed in this PR now that they are tracked in the `RUNTIME_CONFIG` table.